apps/aecore/lib/aecore/tx/transaction.ex

```diff
@@ -7,14 +7,21 @@ defmodule Aecore.Tx.Transaction do
   defmacro __using__(_) do
     quote location: :keep do
       alias Aecore.Tx.Transaction
+      alias Aecore.Governance.GovernanceConstants
+
       @behaviour Transaction
 
       @spec chainstate_senders?() :: boolean()
       def chainstate_senders?() do
         false
       end
 
-      defoverridable chainstate_senders?: 0
+      @spec gas :: non_neg_integer()
```

Comment:
You could probably document here what the `gas` function returns.

Reply:
returning the limit as gas price for each transaction doesn't seem to make sense